src/lib.rs

```diff
@@ -51,25 +52,115 @@ impl RadioPayloadMessage {
     }
 }
 
-// Maybe move to SDK
-lazy_static! {
-    pub static ref SUPPORTED_NETWORK_INTERVALS: HashMap<&'static str, u64> = [
-        ("goerli", 2),
-        ("mainnet", 10),
-        ("gnosis", 5),
-        ("hardhat", 5),
-        ("arbitrum-one", 5),
-        ("arbitrum-goerli", 5),
-        ("avalanche", 5),
-        ("polygon", 5),
-        ("celo", 5),
-        ("optimism", 5),
-    ]
-    .iter()
-    .cloned()
-    .collect();
+#[derive(Debug, Clone)]
+pub struct Network {
+    pub name: NetworkName,
+    pub interval: u64,
+}
+
+pub struct BlockClock {
```

Comment:
Later on maybe we can add `BlockClock` as a mutable field in the `Network` struct